oak_loader/src/options.rs

```diff
@@ -318,7 +313,7 @@ fn create_http_config(opt: &Opt) -> anyhow::Result<oak_runtime::HttpConfiguratio
 
 /// If `oak_debug` is enabled, read root TLS certificate from the specified file. Otherwise, return
 /// the default root TLS certificate from the embedded byte array.
```

Comment:
Could you clarify what format the resulting certificate is in? Is it PEM? base64 encoded?

Reply:
Done.